src/python/pose_format/utils/holistic.py

```diff
@@ -121,3 +122,56 @@ def load_holistic(frames: list, fps: float = 24, width=1000, height=1000, depth=
                                            additional_holistic_config)
 
     return Pose(header, body)
+
+
+def formatted_holistic_pose(width: int, height: int):
+    dimensions = PoseHeaderDimensions(width=width, height=height, depth=1000)
+    header = PoseHeader(version=0.1, dimensions=dimensions, components=holistic_components("XYZC", 10))
```

Comment:
`components` actually includes 5 components, we need to remove the POSE_WORLD_LANDMARKS from it

Reply:
For the shared task data, we assume using:

```
pose.get_components(["POSE_LANDMARKS", "FACE_LANDMARKS", "LEFT_HAND_LANDMARKS", "RIGHT_HAND_LANDMARKS"],
                               {"FACE_LANDMARKS": FACEMESH_CONTOURS_POINTS})
```

which removes the `POSE_WORLD_LANDMARKS `.